README.md

```diff
@@ -1,2 +1,15 @@
 # processing
-Examples about running the STEMMUS-SCOPE model
+Example notebooks showing how to launch the STEMMUS-SCOPE model.
+
+The whole workflow is executed using python and all the required packages are listed in the [`environment.yml`](https://github.com/EcoExtreML/processing/blob/main/environment.yml) file. Please use [`environment.yml`](https://github.com/EcoExtreML/processing/blob/main/environment.yml) to configure your conda environment. For more details about how to setup your conda environment, check [here](https://github.com/EcoExtreML/STEMMUS_SCOPE/tree/main/utils/csv_to_nc) (see steps 1 and 2).
```

Comment:
```suggestion
The  workflow is executed using python and MATLAB Runtime. The python packages are listed in the [`environment.yml`](https://github.com/EcoExtreML/processing/blob/main/environment.yml) file. Follow the instructions below to create conda environment and install MATLAB Runtime.
<details>
  <summary>Create conda environment </summary>
```sh  
# Download and install Conda
wget https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-pypy3-Linux-x86_64.sh
bash Mambaforge-pypy3-Linux-x86_64.sh -b -p ~/mamba

# Update base environment
. ~/mamba/bin/activate
mamba update --name base mamba

# Clone this repository
git clone https://github.com/EcoExtreML/processing.git

# Create a conda environment called 'stemmus' with all required dependencies
cd processing
mamba env create

# The environment can be activated with
. ~/mamba/bin/activate stemmus
```
</details>

<details>
  <summary>Install MATLAB Runtime </summary>
To run the STEMMUS_SCOPE, you need MATLAB Runtime version `2021a`. Use this [download link](https://nl.mathworks.com/products/compiler/matlab-runtime.html) and follow [this instruction](https://nl.mathworks.com/help/compiler/install-the-matlab-runtime.html) to install it.  
Please note MATLAB Runtime is already installed on Snellius, see the script [`run_jupyter_lab_on_compute_node.sh`](https://github.com/EcoExtreML/processing/blob/main/run_jupyter_lab_on_compute_node.sh) on how to load the module.
</details>
```

Reply:
Let's explain it in full and not use the STEMMUS_SCOPE repository to give the references as it is a private repo. I'm not sure if the markdown syntaxes like `<details>` are correct. Could you please check it? thanks.

Reply:
lets add something like this for MATLAB Runtime too:

<details>
  <summary>Install MATLAB Runtime </summary>
To run the STEMMUS_SCOPE, you need MATLAB Runtime version `2021a`. Use this [download link](https://nl.mathworks.com/products/compiler/matlab-runtime.html) and follow [this instruction](https://nl.mathworks.com/help/compiler/install-the-matlab-runtime.html) to install it.  
Please note MATLAB Runtime is already installed on Snellius, see the script [`run_jupyter_lab_on_compute_node.sh`](https://github.com/EcoExtreML/processing/blob/main/run_jupyter_lab_on_compute_node.sh) on how to load the module.
</details>

Reply:
Thanks for your comment! I will add these changes to the readme.